drivers/video/mt9m114.c

```diff
@@ -381,6 +383,46 @@ static int mt9m114_set_output_format(const struct device *dev, int pixel_format)
 	return ret;
 }
 
+static int mt9m114_set_horizontal_mirror(const struct device *dev, int enable)
+{
+	int ret = 0;
+	uint16_t reg = 0;
+
+	ret = mt9m114_read_reg(dev, MT9M114_CAM_SENSOR_CTRL_READ_MODE, sizeof(reg),
+				&reg);
+
+	if (enable) {
+		reg |= BIT(0);
+	} else {
+		reg &= ~BIT(0);
+	}
+
+	ret |= mt9m114_write_reg(dev, MT9M114_CAM_SENSOR_CTRL_READ_MODE, sizeof(reg),
+				&reg);
```

Comment:
Same proposal as below to turn this into `ret = ` and immediately checking the error, which is a bit more verbose but helps avoiding subtle bugs in applications (i.e. checking if some operation is not supported (no point in retrying), or just some failure on I/O (a ladybug accidentally shorted the `I2C.SDA` and `GND` lines, ladybug forgiven and new attempt issued).